kratos/python_scripts/analysis_stage.py

```diff
@@ -125,6 +125,9 @@ def Finalize(self):
 
         KratosMultiphysics.Logger.PrintInfo(self._GetSimulationName(), "Analysis -END- ")
 
+    def GetFinalData(self):
+        return {}

```

Comment:
Sorry for the late request, but can you please add some documentation?

For this function the intended purpose/usage would be very helpful!

Reply:
Done.